idd/Energy+.idd.in

```diff
@@ -6850,10 +6849,31 @@ MaterialProperty:MoisturePenetrationDepth:Settings,
        \required-field
        \units dimensionless
        \type real
-  N5 ; \field Moisture Equation Coefficient d
+  N5 , \field Moisture Equation Coefficient d
        \required-field
        \units dimensionless
        \type real
+  N6 , \field Surface-layer penetration depth
```

Comment:
@nmerket 

Re: First comment from @Myoldmopar, \note should be:
\Ratio of water vapor permeability of stagnant air to water vapor permeability of material